addons/l10n_es_edi_facturae/models/account_move.py

```diff
@@ -170,6 +170,28 @@ def _l10n_es_edi_facturae_convert_computed_tax_to_template(self, computed_tax_di
             },
         }
 
+    def _l10n_es_edi_facturae_convert_payment_terms_to_installments(self):
+        """
+        Convert the payments terms to a list of <Installment> elements to be used in the
+        <PaymentDetails> node of the Facturae XML generation.
+
+        For now we only use the hardcoded '04' value (Credit Transfer).
+        """
+        self.ensure_one()
+        installments = []
+        if self.is_inbound() and self.partner_bank_id:
+            for payment_term in self.line_ids.filtered(lambda l: l.display_type == 'payment_term').sorted('date_maturity'):
+                installments.append({
+                    'InstallmentDueDate': payment_term.date_maturity,
+                    'InstallmentAmount': payment_term.amount_residual_currency, # TODO DYKI: Confirm currency
```

Comment:
@chklop I couldn't find anywhere whether we should use the amount here in the company currency or document currency. Do you have any idea?

Reply:
@chklop Yes, but I mean that I don't know what currency is expected in the facturae XML file. Or do you mean that you think it should be document currency in the facturae file?